src/earn/EarnEntrypoint.tsx

```diff
@@ -28,20 +58,30 @@ export default function EarnEntrypoint() {
         borderRadius={8}
         style={styles.touchable}
         onPress={() => {
-          AppAnalytics.track(EarnEvents.earn_entrypoint_press)
-          navigate(Screens.EarnInfoScreen)
+          AppAnalytics.track(EarnEvents.earn_entrypoint_press, {
+            hasSuppliedPools: hasSuppliedPools,
```

Comment:
```suggestion
            hasSuppliedPools,
```

Reply:
updated in [842c45f](https://github.com/valora-inc/wallet/pull/6233/commits/842c45f19c140afa5d33c7af357b9209f3ce997f)